### PR TITLE
fix(promql): synch generated code

### LINKED ISSUE #13955: promql: running go generate will modify the file and fail CI
Using `go generate` should produce the file that is committed. This appears to not be true for the `query/promql` package. Fix it so the modifications made will remain permanent if regenerated.

### PR DESCRIPTION
**Pull request by affo**
Closes #13955 

The change was not detected by simply running `make`.

I had to `make clean` and then `make`. Is this the expected behavior?

Describe your proposed changes here.

- [ ] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [ ] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [ ] Rebased/mergeable
- [ ] Tests pass
- [ ] http/swagger.yml updated (if modified Go structs or API)
- [ ] Documentation updated or issue created (provide link to issue/pr)
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

**Comment by jsternberg** [comment]
The `make clean` followed by `make` is expected behavior.

File: query/promql/promql.go [hunk 1]
```diff
@@ -3,8 +3,6 @@
 // Package promql implements a promql parser to build flux query specifications from promql.
 package promql
 
-//lint:file-ignore SA6001 Ignore all unused code, it's generated
```
Comment (jsternberg) [inline query/promql/promql.go]: I think you need to find some way to add this lint line.

Comment (affo) [reply query/promql/promql.go]: Thank you

File: query/promql/promql.go [hunk 2]
```diff
@@ -23,42 +21,44 @@ import (
 
 // DO NOT EDIT: This file is auto generated by the pigeon PEG parser generator.
 
+//lint:file-ignore SA6001 Ignore all unused code, it's generated
```
Comment (jsternberg) [inline query/promql/promql.go]: Wow it did not occur to me that you could just do this but it's so obvious it hurts me to see that I didn't realize it.

**Pull request merged**